app/products/agents/actions/websocket/index.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import streamingStore from '@agents/store/streaming_store';
+
+import type {PostUpdateWebsocketMessage} from '@agents/types';
```

Comment:
not a blocker, but I see no reason for this import if we rename the file to `types/index.d.ts` as there are no imports in that file